fcrepo4-exts/migration-utils
# Fcrepo 3054  Implements createPlaceholder in the OCFL Fedora Client

**Implements createPlaceholder in the OCFL Fedora Client**
* * *

**JIRA Ticket**:
https://jira.duraspace.org/browse/FCREPO-3054

# What does this Pull Request do?
Implements createPlaceholder in the OCFL Fedora Client allowing resource placegolders to be created within the OCFL storage root

# What's new?
Interface implementation of createPlaceholder method and associated unit and integration tests

# How should this be tested?

* attempt to use the OCFLFedora4Client to create placeholders and check that they appear on storage

# Interested parties
@awoods  @fcrepo4/committers
